docs/interest-groups.html

```diff
@@ -44,18 +44,14 @@ <h3>Currently Joined</h3>
 			import * as idb from 'https://cdn.jsdelivr.net/npm/idb-keyval@5/dist/esm/index.js';
 			import { Fledge } from './scripts/api/esm/index.js';
 			// const fledge = new Fledge('http://localhost:8000/iframe.html');
-			const fledge = new Fledge('https://magniteengineering.github.io/fledge.polyfill/iframe.html');
+			const fledge = new Fledge('https://ux.rubiconproject.com/fledge/iframe.html');
```

Comment:
don't love adding this url into the repo, I thought we were just using for your demo. also is it still necessary given we found the *real* issue?